packages/testing/src/testing.ts

```diff
@@ -89,7 +89,7 @@ export function createTestingPinia({
 
   plugins.forEach((plugin) => pinia.use(plugin))
 
-  const createSpy = _createSpy || (typeof jest !== 'undefined' && jest.fn)
+  const createSpy = _createSpy || (typeof jest !== 'undefined' && jest.fn) || (typeof vitest !== 'undefined' && vitest.fn)
```

Comment:
Isn't it 
```suggestion
  const createSpy = _createSpy || (typeof jest !== 'undefined' && jest.fn) || (typeof vi !== 'undefined' && vi.fn)
```

Reply:
Both of them are valid: https://github.com/vitest-dev/vitest/blob/main/packages/vitest/globals.d.ts